reference/ToDo/src/ToDo/Views/HomePage.xaml

```diff
@@ -1,143 +1,165 @@
-								Value="TaskList" />
-					</VisualState.Setters>
-				</VisualState>
-			</VisualStateGroup>
-		</VisualStateManager.VisualStateGroups>
+    <Style x:Key="DefaultTaskListPathIconStyle"
+           TargetType="PathIcon">
+      <Setter Property="Foreground"
+              Value="{ThemeResource PrimaryVariantLightBrush}" />
+      <Setter Property="Data"
+              Value="{StaticResource Icon_List}" />
+    </Style>
 
-		<muxc:NavigationView x:Name="NavView"
-							 MenuItemsSource="{Binding WellKnownLists}"
-							 FooterMenuItemsSource="{Binding CustomLists}"
-							 AlwaysShowHeader="True"
-							 PaneDisplayMode="Left"
-							 OpenPaneLength="{StaticResource SplitModeSidePaneWidth}"
-							 IsPaneToggleButtonVisible="False"
-							 IsSettingsVisible="False"
-							 Style="{StaticResource HomeNavigationViewStyle}"
-							 CompactModeThresholdWidth="0"
-							 ExpandedModeThresholdWidth="0">
+    <Style x:Key="TaskListPathIconStyle"
+           TargetType="PathIcon">
+      <Setter Property="Foreground"
+              Value="{ThemeResource PrimaryVariantDarkBrush}" />
+      <Setter Property="Data"
+              Value="{StaticResource Icon_House_Siding}" />
+    </Style>
 
-			<!-- Header: Profile (Setting button) + Search button -->
-			<muxc:NavigationView.PaneHeader>
-				<Grid Grid.Row="0"
-					  Padding="16,8"
-					  ColumnSpacing="8"
-					  ColumnDefinitions="*,Auto">
+    <Style x:Key="ImportantTaskListPathIconStyle"
+           TargetType="PathIcon">
+      <Setter Property="Foreground"
+              Value="{ThemeResource ErrorBrush}" />
+      <Setter Property="Data"
+              Value="{StaticResource Icon_Star_Rate}" />
+    </Style>
 
-					<Button Grid.Column="0"
-							uen:Navigation.Request="!Settings"
-							BorderThickness="0"
-							HorizontalAlignment="Stretch"
-							HorizontalContentAlignment="Stretch"
-							Style="{StaticResource OutlinedButtonStyle}">
-						<Grid ColumnDefinitions="Auto,*"
-							  ColumnSpacing="8">
-							<PersonPicture Grid.Column="0"
-								   DisplayName="{Binding CurrentUser.Name}"
-								   ProfilePicture="{Binding ProfilePicture, Converter={StaticResource  BitmapSourceConverter}}"
-								   Width="40"
-								   Height="40"
-										   VerticalAlignment="Center" />
-							<TextBlock Grid.Column="1"
-									   Text="{Binding CurrentUser.Name}"
-									   VerticalAlignment="Center"
-									   Style="{StaticResource TitleMedium}" />
-						</Grid>
-					</Button>
+    <converters:TaskListToValueConverter x:Key="TaskListPathIconStyleConverter"
+                                         ImportantValue="{StaticResource ImportantTaskListPathIconStyle}"
+                                         TasksValue="{StaticResource TaskListPathIconStyle}"
+                                         DefaultValue="{StaticResource DefaultTaskListPathIconStyle}" />
+  </Page.Resources>
+  <Grid uut:VisibleBoundsPadding.PaddingMask="All">
```

Comment:
We should also switch this to use SafeArea eventually. Doesn't have to be with this PR

Reply:
Yeah I will deal with that one at the same time we are dealing with the iOS safe area issue